Adyen/UI/Form/FormItemViewBuilder.swift

```diff
@@ -59,4 +59,16 @@ public struct FormItemViewBuilder {
     public func build(with item: FormSeparatorItem) -> FormItemView<FormSeparatorItem> {
         FormSeparatorItemView(item: item)
     }
+
+    /// Builds `FormSeparatorItemView` from `FormSeparatorItem`.
+    /// :nodoc:
+    public func build(with item: FormFoldableItem) -> FormItemView<FormFoldableItem> {
+        FormFoldableItemView(item: item)
```

Comment:
I prefer `FoldableFormItemView`, the parent class name should have the subclass qualifier at the beginning:

```
parent class: FormItemView
subclass: FoldableFormItemView
```

```
parent class: ParentClass
subclass: ChildParentClass
```

Reply:
We can rename, but this would be a breaking change.
Let's address it in another PR